src/Scan.js

```diff
@@ -178,6 +178,8 @@ class Scan extends React.Component {
   }
 
   checkIn = () => {
+    if (this.state.loading) return undefined;
```

Comment:
`if (this.state.loading) return undefined;`

can be simplified to:

`if (this.state.loading) return;`

I'm not sure that undefined should be used in this case. I would prefer to prevent using undefined assignment and use null in this case.

Reply:
Null is also does not make sense. For me, more natural is to return `undefined` as it is what is used when return without anything is used. I do not like this eslint rule, but do not want to add eslint ignore statement for just that.